test: AsyncResultSetImplStressTest timed out on Windows build

`AsyncResultSetImplStressTest` timed out on Windows build.

```
2021-05-24T07:13:01.5068744Z [INFO] Running com.google.cloud.spanner.AsyncResultSetImplStressTest
2021-05-24T07:15:56.5980265Z [ERROR] Tests run: 28, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 175.074 s <<< FAILURE! - in com.google.cloud.spanner.AsyncResultSetImplStressTest
2021-05-24T07:15:56.5984627Z [ERROR] com.google.cloud.spanner.AsyncResultSetImplStressTest.asyncToList[rows = 10]  Time elapsed: 120.524 s  <<< ERROR!
2021-05-24T07:15:56.5988089Z org.junit.runners.model.TestTimedOutException: test timed out after 120 seconds
2021-05-24T07:15:56.6001321Z 	at com.google.cloud.spanner.AsyncResultSetImplStressTest.asyncToList(AsyncResultSetImplStressTest.java:214)
2021-05-24T07:15:56.6003591Z 
```

--- a/google-cloud-spanner/src/test/java/com/google/cloud/spanner/AsyncResultSetImplStressTest.java
+++ b/google-cloud-spanner/src/test/java/com/google/cloud/spanner/AsyncResultSetImplStressTest.java
@@ -56,7 +56,7 @@ public class AsyncResultSetImplStressTest {
   private static final int TEST_RUNS = 25;
 
   /** Timeout is applied to each test case individually. */
-  @Rule public Timeout timeout = new Timeout(120, TimeUnit.SECONDS);
+  @Rule public Timeout timeout = new Timeout(240, TimeUnit.SECONDS);
 
   @Parameter(0)
   public int resultSetSize;

